src/Term.ts

```diff
@@ -38,7 +39,10 @@ class Term {
       });
     }
 
-    const status = await exec("npx size-limit --json", [], {
+    const status = await exec(`${runner} size-limit --json`, [], {
+      env: {
+        GITHUB_ACTIONS: ""
+      },
```

Comment:
Yarn tries to detect the CI environment and provide special formatting to stdout: https://github.com/yarnpkg/berry/blob/b6f59ef07394533be68bc7b3b60e10428b6ae0c2/packages/yarnpkg-core/sources/StreamReport.ts#L28-L34. This interferes with the JSON parsing that the action performs, so I'm disabling it with this code